benchmarks/CMakeLists.txt

```diff
@@ -25,38 +25,39 @@ set(_benchmarks gcl_test_halo_exchange_3D_generic_full comm_2_test_halo_exchange
 set(_benchmarks_mt )
 
 foreach (_t ${_benchmarks})
-  add_executable(${_t} ${_t}.cpp)
-  target_link_libraries(${_t} MPI::MPI_CXX GridTools::gridtools gtest_main_bench ghexlib)
+    add_executable(${_t} ${_t}.cpp)
+    target_link_libraries(${_t} gtest_main_bench)
 
-  add_executable(${_t}_1_pattern ${_t}.cpp)
-  target_compile_definitions(${_t}_1_pattern PUBLIC GHEX_1_PATTERN_BENCHMARK)
-  target_link_libraries(${_t}_1_pattern MPI::MPI_CXX GridTools::gridtools gtest_main_bench ghexlib)
+    add_executable(${_t}_1_pattern ${_t}.cpp)
+    target_compile_definitions(${_t}_1_pattern PUBLIC GHEX_1_PATTERN_BENCHMARK GCL_MPI) 
+    target_link_libraries(${_t}_1_pattern gtest_main_bench)
```

Comment:
incorrect indentation (I usually use two spaces, which seems to be the convention now; I would really stick to it).